EIPS/eip-2537.md

```diff
@@ -0,0 +1,293 @@
+---
+eip: 2537
+title: BLS12-381 curve operations
+author: Alex Vlasov (@shamatar)
+discussions-to: https://github.com/ethereum/EIPs/pull/2537
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-02-21
+requires : 1109, 2046
```

Comment:
```suggestion
requires  1109, 2046
```

Reply:
But actually I think these are not technical prerequisites.

Reply:
I wanted to label them optional, but CI validator didn’t allow me to.

Reply:
There is no optionality in that field :)